tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/IQueryRequest.cs

```diff
@@ -3,12 +3,14 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+using Datadog.Trace.DuckTyping;
+
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate
 {
     /// <summary>
     /// HotChocolate.Execution.IQueryRequest interface for ducktyping
     /// </summary>
-    internal interface IQueryRequest
+    internal interface IQueryRequest : IDuckType
```

Comment:
Could we enable nullability for this one? Seeing as you've already done the analysis, it would be good to make _all_ these types nullable aware 🙂